test/unit/oatbox/user/UserLanguageServiceTest.php

```diff
@@ -73,7 +115,7 @@ public function testIsDataLanguageEnabled()
         $service = $this->getService();
         $this->assertEquals(true, $service->isDataLanguageEnabled());
 
-        $service = $this->getService([
+        $service = new UserLanguageService([
```

Comment:
You can use the previous method.

```suggestion
        $service = $this->getService([
```

Reply:
I missed that one while rebasing with latest changes from develop, but I've already changed it again on 876a8d453f0b1f95ffd03db8ebc9305e855d8447 :ok_hand: